modules/nf-core/nanoq/main.nf

```diff
@@ -0,0 +1,74 @@
+process NANOQ {
+
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+
+    container "${workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/nanoq:0.10.0--h031d066_2' :
+        'biocontainers/nanoq:0.10.0--h031d066_2'}"
+
+    input:
+    tuple val(meta), path(ontreads)
+    val(output_type) // u: uncompressed; b: Bzip2; g: Gzip; l: Lzma
+
+    output:
+    tuple val(meta), path("*.stats")                                                  , emit: stats
+    tuple val(meta), path("*")                                                        , emit: reads
```

Comment:
Yes, this is too broad and capturing the stats and versions as well.

Reply:
Right, that's why all three files were captured by content 1.